tests/fixtures/pretty.sql

```diff
@@ -99,14 +99,14 @@ WITH cte1 AS (
       FOO(CASE WHEN a AND b THEN c AND d ELSE 3 END)
     GROUP BY
       x,
-    GROUPING SETS (
-      a,
-      (b, c)
-    ),
-    CUBE (
-      y,
-      z
-    )
+      GROUPING SETS (
```

Comment:
why did the indentation change?

Reply:
I think I started with the same indentation but later changed it because `CUBE`, `ROLLUP` and `GROUPING SETS` are expressions corresponding to `GROUP BY` and not "query clauses" like `ORDER BY`, `HAVING` etc, which are currently in the same level as `SELECT`. It made more sense to indent them.